packages/project/scripts/validateIsolated.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * This is a stripped down self-contained example of validating V4 import files
+ * without reliance on Nori's errors library or the custom validationRule that
+ * enforces crop events being in the expected crop year.
+ */
+
+import * as fs from 'fs';
+import { inspect } from 'util';
+import Ajv from 'ajv';
+// Adds `date` type format support
+import addFormats from 'ajv-formats';
+import type { ErrorObject } from 'ajv';
+
+import { Project } from '../src/v4-specification';
+import * as schema from '../src/json/v4-specification.json';
+
+export const validateProjectData = (
+  data: Project
+): {
+  valid: boolean;
+  message?: string;
+  errors?: ErrorObject[];
+} => {
+  const ajv = addFormats(
+    new Ajv({
+      useDefaults: 'empty',
+      allErrors: true,
+      inlineRefs: false,
+      $data: true,
+      allowUnionTypes: true,
+    })
+  );
+  // Ignore the custom NORI errorMessage and validationRules clauses
```

Comment:
might be nice to just `console.warn("Note that this validation library does not currently validate the `validationRule` or `errorMessage` fields");` for external consumers